modules/intensity_transform/test/test_intensity_transform.cpp

```diff
@@ -0,0 +1,199 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+TEST(intensity_transform_logTransform, accuracy)
+{
+    cv::Mat const image = (cv::Mat_<uchar>(10, 10) <<
```

Comment:
Could you please avoid using of this initializer in OpenCV? Consider using RAW array instead.
see https://github.com/opencv/opencv/pull/15959#discussion_r349847572